src/effects/VST3/VST3Effect.cpp

```diff
@@ -967,19 +777,15 @@ void VST3Effect::ExportPresets(const EffectSettings& settings) const
       );
       return;
    }
-
-   if (!StoreSettings(GetSettings(settings)))
-      return;
```

Comment:
The elimination of `StoreSettings` leaves the argument `settings` unused in this function, at least so I see at the fifth commit in the sequence.  So this "breaks" that function, but I presume you fix it in later commits I have not yet reviewed.  I understand from other commit comments that you put back a local wrapper object in the function for this purpose.

Strictly then this commit sequence isn't preserving intended behavior all the way through but I won't insist on that now.

Reply:
Using StoreSettings on VST3Effect instance could change it's state, which isn't correct behaviour. Considering that there is a single instance and the copy of settings that passed to this function was more likely fetched from same instance it shouldn't harm?

Reply:
The point, however, is to make ExportPresets work independently of any instance.  And you achieve that in the later commit with a temporary Wrapper.

Reply:
I consider this resolved but I leave it open so you read and understand my previous comment.